### DISCUSSION TITLE
Keep history of persistence

### DISCUSSION THREAD

**Discussion by rodja** [zauberzeug/rosys#108] [Ideas / Feature Requests]
It would be great if we would save the latest five or so versions of any persistence. Sometimes due to some code change a calibration or similar gets erased. Instead of doing a labor intensive re-calibration it would be great to just use an copy an old version of the storage. It must not be restorable from the UI but simply by terminal.

**Comment by codingpaula** [comment]
minimal implementation was done here: PR #140
